app/containers/Bookings/Details/index.scss

```diff
@@ -17,9 +17,10 @@
 .image-container {
   height: 400px;
   margin-top: 6%;
-  width: 320px;
+  width: 240px;
```

Comment:
Are the images always the same size?  I would have thought that if someone had previously specified those different dimensions then there must have been images that size too?

Reply:
I wasn't able to determine whether the images were all the same or of different sizes as there was nothing in the DB to tell me.